package.json

```
@@ -32,6 +32,7 @@
     "karma-mocha": "^2.0.1",
     "karma-mocha-reporter": "^2.2.5",
     "mocha": "^9.1.1",
+    "puppeteer": "^12.0.1",
```

If we installed `playwright` instead, it'll also install firefox, chrome and webkit which means we could test in all 3!

```suggestion
    "playwright": "^1.17.0",
```